pyriemann/datasets/simulated.py

```diff
@@ -78,8 +79,8 @@ def make_masks(n_masks, n_dim0, n_dim1_min, rs):
 
 
 def make_gaussian_blobs(n_matrices=100, n_dim=2, class_sep=1.0, class_disp=1.0,
-                        return_centers=False, random_state=None, *,
-                        mat_mean=.0, mat_std=1., n_jobs=1,
+                        return_centers=False, center_dataset=True,
+                        random_state=None, input_centers=None, *, n_jobs=1,
```

Comment:
This modif breaks API...

Reply:
I'm not sure I understand the problem. Yeah, I took the `mat_mean` and `mat_std` out of the code but thought I had done it in a clean way so to not have any conflicts and breaks. You think it would be necessary to keep them?

Reply:
I'm not sure I understand the problem. Yeah, I took the mat_mean and mat_std out of the code but thought I had done it in a clean way so to not have any conflicts and breaks. You think it would be necessary to keep them?

Reply:
These parameters are keywords only, so I think it's ok.
I prefer `centers` argument, because without it, I was unable to generate an appropriate dataset for #200.